egs/polyphone_swiss_french/asr1/run.sh

```diff
@@ -0,0 +1,225 @@
+#!/bin/bash
+
+# Copyright 2017 Johns Hopkins University (Shinji Watanabe)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+. ./path.sh || exit 1;
+. ./cmd.sh || exit 1;
+
+# general configuration
+backend=pytorch
+stage=-1       # start from 0 if you need to start from data preparation
+stop_stage=100
+ngpu=4         # number of gpus ("0" uses cpu, otherwise use gpu)
+debugmode=1
+dumpdir=dump   # directory to dump full features
+N=0            # number of minibatches to be used (mainly for debugging). "0" uses all minibatches.
+verbose=0      # verbose option
+
+# feature configuration
+do_delta=false
+
+preprocess_config=conf/specaug.yaml
+train_config=conf/train.yaml
+lm_config=conf/lm.yaml
+decode_config=conf/decode.yaml
+dataprep_config=conf/dataprep.yaml
+
+# rnnlm related
+lm_resume=        # specify a snapshot file to resume LM training
+lmtag=            # tag for managing LMs
+
+# decoding parameter
+recog_model=model.acc.best # set a model to be used for decoding: 'model.acc.best' or 'model.loss.best'
+n_average=10
+
+# bpemode (unigram or bpe)
+nbpe=800
+bpemode=unigram
+
+# exp tag
+tag="specaug" # tag for managing experiments.
```

Comment:
We can make a tag variable blank and add specaug related tag through the `preprocess_conf` variable, e.g., 
https://github.com/espnet/espnet/blob/master/egs/librispeech/asr1/run.sh#L203-L213